packages/sampler/stories/default/VirtualGridList.js

```diff
@@ -61,27 +61,38 @@ export default {
 	component: 'VirtualGridList'
 };
 
-export const VirtualListVirtualGridList = () => (
+export const VirtualListVirtualGridList = (args) => (
```

Comment:
How about _VirtualGridList?

Reply:
That seems better than before. Thank you!